tesseract_core/sdk/docker_client.py

```diff
@@ -666,22 +674,17 @@ def up(compose_fpath: str, project_name: str) -> str:
             )
             return project_name
         except subprocess.CalledProcessError as ex:
+            logger.error(str(ex))
+            logger.error(ex.stderr.decode())
             # If the project successfully started, try to get the logs from the containers
             project_containers = Compose.list(include_stopped=True).get(
-                project_name, None
+                project_name, ()
             )
-            if project_containers:
-                container = Containers.get(project_containers[0])
-                stderr = container.logs(stderr=True)
-                raise ContainerError(
-                    f"Failed to start Tesseract container: {container.name}, logs: ",
-                    stderr,
-                ) from ex
-            logger.error(str(ex))
-            logger.error(ex.stderr.decode())
-            raise ContainerError(
-                "Failed to start Tesseract containers.", ex.stderr
-            ) from ex
+            for container_name in project_containers:
+                container = Containers.get(container_name)
+                logger.error(f"Container {container_name} logs:")
+                logger.error(indent(container.logs(stderr=True).decode(), " > "))
```

Comment:
Drive-by change, the old logic only printed container logs, not the actual exception from the CLI which masked errors in my tests.